bindings/python/iota_sdk/types/block/metadata.py

```diff
@@ -1,36 +1,10 @@
 # Copyright 2023 IOTA Stiftung
 # SPDX-License-Identifier: Apache-2.0
 
-from __future__ import annotations
 from enum import Enum, IntEnum
-from dataclasses import dataclass
-from typing import Optional
 
-from iota_sdk.types.common import json
-from iota_sdk.types.block.id import BlockId
-from iota_sdk.types.transaction_metadata import TransactionFailureReason, TransactionState
 
-
-@json
-@dataclass
-class BlockMetadata:
```

Comment:
Why?

Reply:
There is BlockMetadataResponse instead now